vault/blog.2021.2021-09-27-markdown-on-mobile.md

```diff
@@ -0,0 +1,97 @@
+---
+id: fDCVPEo3guCFWPdxokXHU
+title: 'Mobile apps for notes in Markdown: A comparison'
```

Comment:
When writing a title, think what you would click through as a hacker news article. Would change to

**Best Mobile Note-Taking Apps for Markdown**